inference-engine/tests/functional/shared_test_classes/src/single_layer/fake_quantize.cpp

```diff
@@ -113,4 +114,117 @@ void FakeQuantizeLayerTest::UpdateSeed() {
     std::cout << "\033[0;32m" << "[          ] " << "\033[0;0m"
               << "seed = " << seed << std::endl;
 }
+
+
+
+std::string FakeQuantizeLayerTestRevise::getTestCaseName(testing::TestParamInfo<fqLayerTestParamsSetRevise> obj) {
```

Comment:
I think it can be changed to 
```suggestion
std::string FakeQuantizeLayerTestRevise::getTestCaseName(const testing::TestParamInfo<fqLayerTestParamsSetRevise>& obj) {
```

Reply:
done